pkg/driverbuilder/local.go

```diff
@@ -128,6 +128,7 @@ func (lbp *LocalBuildProcessor) Start(b *builder.Build) error {
 		ctx, cancelFunc := context.WithTimeout(context.Background(), time.Duration(lbp.timeout)*time.Second)
 		defer cancelFunc()
 		cmd := exec.CommandContext(ctx, "/bin/bash", "-c", driverkitScript)
+		cmd.Env = os.Environ()
```

Comment:
If `cmd.Env` is not empty when running the cmd, it won't get auto-filled with `Environ`. Force fill before appending our env map.